packages/runner-shared/src/event-manager.js

```diff
@@ -442,12 +442,13 @@ export const eventManager = {
       reporterBus.emit('reporter:log:state:changed', displayProps)
     })
 
-    Cypress.on('before:screenshot', (config, cb) => {
+    const handleBeforeScreenshot = (config, cb) => {
       const beforeThenCb = () => {
         localBus.emit('before:screenshot', config)
         cb()
       }
 
+      // TODO: Should this come from primary or secondary?
```

Comment:
Right now the env value is from the primary, do we want to pass it in from the secondary?

Reply:
No, this should come from the primary. It's meant to be global. It keeps the reporter command log from rendering at all as well any logic/messaging involved. Users can set this, usually via command line, to improve performance if they don't care about the command log being displayed.